### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by rantahar**
Add the timeline we built in the unconference, with small clarifications.

Review:
 - Typos, language
 - Anything unclear?
 - Forgot anything?
 - What should be added?

### CODE REVIEW

File: content/other/timeline.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
++++
++++
+
+# Upcoming activities
+
+> Note:
+>
+> This is a rough plan of what our activities this in Fall 2021, to be updated
```
Comment (lucaferranti) [inline content/other/timeline.md:8]:
```suggestion
> This is a rough plan of our activities in Fall 2021, to be updated
```

File: content/other/timeline.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+> for the Spring in December. Please let us know if you can suggest things to
```
Comment (lucaferranti) [inline content/other/timeline.md:9]:
```suggestion
> for Spring 2022 in December. Please let us know if you can suggest things to
```

File: content/other/timeline.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+- 2.9 at 14:00 - 16:00 CEST, 15:00 - 17:00 EEST: First Nordic RSE annual meeting ([public calendar](https://calendar.google.com/calendar/embed?src=8li6hjcjm95g76pgte1p5pi05c%40group.calendar.google.com&ctz=Europe%2FStockholm), [public iCal feed](https://calendar.google.com/calendar/ical/8li6hjcjm95g76pgte1p5pi05c%40group.calendar.google.com/public/basic.ics))
+  - Approve statutes
+  - Select board
+  - Authorize the board to register the association
+
+- First board meeting
+  - submit registration form for the association
+
+### October
```
Comment (lucaferranti) [inline content/other/timeline.md:45]: should the international RSE day be mentioned here (14.10.2021)?

File: content/other/timeline.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+### July
+
+- International RSE survey starts.
+  - Nordic RSE will advertise and increase participation in the Nordics.
+- Add this timeline on the Nordic-RSE website.
+- Proposal to change the Finnish ranking of the [Journal of Open Source Software](https://joss.theoj.org/) from [JUFO ranking](https://www.tsv.fi/julkaisufoorumi/haku.php?lang=en) 0 to 1.
+
+### August
+
+- The first event in the Nordic RSE seminar series.
```
Comment (lucaferranti) [inline content/other/timeline.md:26]:
```suggestion
- The first event in the Nordic RSE [seminar series](https://nordic-rse.org/events/seminar-series/).
```

File: content/other/timeline.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- Add some useful links to the website, including:
+  - Hands-on Scientific Computing.
+  - The Aalto RSE career structure.
+  - List of journals publishing open source software.
+
+### September
+
+- Create a how to page for starting an RSE group.
```
Comment (lucaferranti) [inline content/other/timeline.md:34]:
```suggestion
- Create a how-to page for starting an RSE group.
```

File: content/other/timeline.md (hunk 3 above)
Comment (rantahar) [reply content/other/timeline.md:45]: Yes. I'm on vacation but will get back to it. (but feel welcome to add it if you have time)

Comment (rantahar) [reply content/other/timeline.md:45]: I guess that would require merging and creating a new PR.

**Review by PatricHolmvall — APPROVED** [review]
This looks very good, thanks for uploading.

Not sure if it is too "meta" to add that we will also work on improving the page/presentation for the seminar series. (Background: We need to have a better page for the seminar series, with relevant information and tentative schedule that can be filled with events. We got feedback regarding the unconference page that it was too text heavy. We should maybe aim for a page with better use of layout+graphics, to make it more appealing)

**Comment by bast** [comment]
Thanks! Merging. I will add the Oct 14 part in a separate PR.

**Pull request merged**